README.md

```diff
@@ -101,6 +101,15 @@ Whenever your application loads, these variables will be available in `ENV`:
 config.fog_directory  = ENV['S3_BUCKET']
 ```
 
+Comments may be added to your file as such:
+
+```comments
```

Comment:
I've double checked using both 

VAR = "SOMETHING#XXX" notation and 

VAR: 'SOMETHING#XXX" notation